File: Latihan3-php/PC.php
<?php

class PC{
    private $p;
    private $d;
    private $r;
    private $totalPrice;

    public function __construct(){
        $this->p = new Processor();
        $this->d = new Disk();
        $this->r = new RAM();
        $this->totalPrice = 0;
    }

    // Get-set Processor
    public function setProcessor($p){
        $this->p = $p;
    }

    public function getProcessor(){
        return $this->p;
    }

    // Get-set Disk
    public function setDisk($d){
        $this->d = $d;
    }

    public function getDisk(){
        return $this->d;
    }

    // Get-set RAM
    public function setRAM($r){
        $this->r = $r;
    }

    public function getRAM(){
        return $this->r;
    }

    // Get-set totalPrice
    public function setTotalPrice($p1, $p2, $p3){
        $this->totalPrice = $p1 + $p2 + $p3;
    }

    public function getTotalPrice(){
        return $this->totalPrice;
    }
}
?>

File: Latihan3/PC.java
package Latihan3;

class PC {
    private Processor p;
    private Disk d;
    private RAM r;
    private int totalPrice;

    // Constructor
    public PC(){
    }

    // Get-set Processor
    public Processor getProcessor() {
        return p;
    }

    public void setProcessor(Processor p) {
        this.p = p;
    }
    
    // Get-set Disk
    public Disk getDisk() {
        return d;
    }

    public void setDisk(Disk d) {
        this.d = d;
    }
    
    // Get-set RAM
    public RAM getRAM() {
        return r;
    }

    public void setRAM(RAM r) {
        this.r = r;
    }
    
    // Get-set Price
    public int getTotalPrice() {
        return totalPrice;
    }

    public void setTotalPrice(int p1, int p2, int p3) {
        this.totalPrice = p1 + p2 + p3;
    }
}